helm-org.el

```diff
@@ -58,9 +58,11 @@ NOTE: This will be slow on large org buffers."
   (org-show-entry))
 
 (cl-defun helm-source-org-headings-for-files (filenames
-                                              &optional (min-depth 1) (max-depth 8))
+                                              &optional (min-depth 1) (max-depth 8) (parents nil))
   (helm-build-sync-source "Org Headings"
-    :candidates (helm-org-get-candidates filenames min-depth max-depth)
+    :candidates (helm-org-get-candidates filenames min-depth max-depth parents)
+    :candidate-transformer (if parents 'reverse nil)
```

Comment:
This seems like it should go in `helm-org-parent-headings`, because it would make the `if` unnecessary, but when I specified it there as a keyword attribute, it didn't have any effect.  Is there a better way to do this?